pkg/apis/clickhouse.altinity.com/v1/type_config_chop.go

```diff
@@ -324,6 +326,7 @@ type OperatorConfig struct {
 	// 1. Metrics requests
 	// 2. Schema maintenance
 	// User credentials can be specified in additional ClickHouse config files located in `chUsersConfigsPath` folder
+	CHScheme   string `json:"chScheme" yaml:"chScheme"`
```

Comment:
This section is kind of `deprecated` and is kept for backward-compatibility only. That's OK, it's just nice extra.

Reply:
I saw, wasn't sure, i thought it was no harm.